cdap-watchdog/src/main/java/co/cask/cdap/logging/write/AvroFileWriter.java

```diff
@@ -56,18 +59,19 @@
 
   /**
    * Constructs an AvroFileWriter object.
+   * @param cConf the CDAP configuration.
    * @param fileMetaDataManager used to store file meta data.
-   * @param baseDir base dir.
+   * @param rootDir the CDAP root dir on the filesystem
    * @param schema schema of the Avro data to be written.
    * @param maxFileSize Avro files greater than maxFileSize will get rotated.
    * @param syncIntervalBytes the approximate number of uncompressed bytes to write in each block.
    * @param inactiveIntervalMs files that have no data written for more than inactiveIntervalMs will be closed.
    */
-  public AvroFileWriter(FileMetaDataManager fileMetaDataManager,
-                        Location baseDir, Schema schema,
-                        long maxFileSize, int syncIntervalBytes, long inactiveIntervalMs) {
+  public AvroFileWriter(CConfiguration cConf, FileMetaDataManager fileMetaDataManager, Location rootDir,
+                        Schema schema, long maxFileSize, int syncIntervalBytes, long inactiveIntervalMs) {
+    this.cConf = cConf;
```

Comment:
The only reason of having `cConf` here is to get log base dir. Should we just pass log base dir directly here?